### DISCUSSION TITLE
Oxygen tank rework thing

### DISCUSSION THREAD

**Discussion by DiamondWalker** [TeamGalacticraft/Galacticraft#247] [Feature Requests]
In the old galacticraft oxygen tanks are divided into three tiers. The "heavy" oxygen tanks can hold three times the amount of oxygen as the "light" oxygen tanks. The thing is, all of them are about as difficult to craft. In my experience, there is absolutely no reason whatsoever to use anything but the heavy tanks (unless you're _really_ lazy; too lazy to be playing this mod). Instead, the tanks should require metals from the different bodies:
Light - Steel
Medium - Meteoric Iron
Heavy - Titanium

Additionally, the wool color should be unrelated to the tank "size". I personally really like mixing two different tank colors, and it's really annoying not being able to do that without losing oxygen capacity. I believe it would be better if you could use wool of any color (like beds) while the canister itself matches the color of the metal used.

**Comment by JunePrimavera** [comment]
I like the wool colour thing, but I feel like this would just be annoying rather than something to work for, unless tweaks were made (Maybe some kind of "upgrades") since the light one is so small (I always ran out very quickly when using it)

Maybe added tiers, like ultra heavy or something may be more suited?

**Reply by DiamondWalker** [reply]
> I like the wool colour thing, but I feel like this would just be annoying rather than something to work for, unless tweaks were made (Maybe some kind of "upgrades") since the light one is so small (I always ran out very quickly when using it)
> 
> Maybe added tiers, like ultra heavy or something may be more suited?

It's worth noting medium holds a nice amount of oxygen and meteoric iron can be obtained very easily once you get to the moon, but I see what you mean, especially for newer players who need time to figure out how to set up their oxygen.

Maybe they could all be moved 1 tier up? Light goes from 900 to 1800, medium from 1800 to 2700, and heavy from 2700 to 3600. I think that would be a nice way to balance it. I don't think we need more tiers, since the point is making the ones we have now more relevant.

**Reply by JunePrimavera** [reply]
> > I like the wool colour thing, but I feel like this would just be annoying rather than something to work for, unless tweaks were made (Maybe some kind of "upgrades") since the light one is so small (I always ran out very quickly when using it)
> > Maybe added tiers, like ultra heavy or something may be more suited?
> 
> It's worth noting medium holds a nice amount of oxygen and meteoric iron can be obtained very easily once you get to the moon, but I see what you mean, especially for newer players who need time to figure out how to set up their oxygen.
> 
> Maybe they could all be moved 1 tier up? Light goes from 900 to 1800, medium from 1800 to 2700, and heavy from 2700 to 3600. I think that would be a nice way to balance it. I don't think we need more tiers, since the point is making the ones we have now more relevant.

Yeah, moving them up sounds better better than more tiers. a much simpler solution lol.
I like the idea
